e2e-tests/portal/cypress/support/portal/createBssApplicationReadyForCheck.js

```diff
@@ -0,0 +1,81 @@
+const dashboard = require('../../e2e/pages/dashboard');
+const selectScheme = require('../../e2e/pages/selectScheme');
+const beforeYouStart = require('../../e2e/pages/beforeYouStart');
+const bankDetails = require('../../e2e/pages/bankDetails');
+const contract = require('../../e2e/pages/contract');
+const aboutSupplier = require('../../e2e/pages/contract/about/about-supplier');
+const aboutBuyer = require('../../e2e/pages/contract/about/about-buyer');
+const aboutFinancial = require('../../e2e/pages/contract/about/about-financial');
+const eligibilityCriteria = require('../../e2e/pages/eligibilityCriteria');
+const eligibilityPreview = require('../../e2e/pages/eligibilityPreview');
+const bondDetails = require('../../e2e/pages/bondDetails');
+const bondFinancialDetails = require('../../e2e/pages/bondFinancialDetails');
+const bondFeeDetails = require('../../e2e/pages/bondFeeDetails');
+const contractReadyForReview = require('../../e2e/pages/contract/contract-readyForReview');
+const login = require('./login');
+const MOCK_USERS = require('../../../../e2e-fixtures/index');
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+const createBssApplicationReadyForCheck = () => {
```

Comment:
documentation?

Reply:
Sorry, what documentation is required?

Reply:
Something like this?

```js
/**
 * Create a BSS/EWCS deal via the UI with a "ready for check" status
 */
```

Does the command need to return the deal?

Reply:
Also, should we call this `createBssDealReadyForCheck` instead of `createBssApplicationReadyForCheck`?

Reply:
Added documentation and changed from `createBssApplicationReadyForCheck` to `createBssDealReadyForCheck`.